src/css/cards.less

```diff
@@ -1,3 +0,0 @@
-.__mobiledoc-card {
-  display: inline-block;
```

Comment:
@bantic What I mean is to be able to pick which cards are `inline-block`. Currently I can't target individual card containers, which makes impossible to, for example, have two cards, of which one is 100% wide and the other isn't.